tests/app/dao/test_ft_billing_dao.py

```diff
@@ -856,6 +857,84 @@ def test_fetch_daily_volumes_for_platform(
     assert results[1].letter_sheet_totals == 40
 
 
+def test_fetch_daily_sms_provider_volumes_for_platform_for_platform_groups_values_by_provider(
+    notify_db_session,
+):
+    services = [
+        create_service(service_name='a'),
+        create_service(service_name='b')
+    ]
+    templates = [
+        create_template(services[0]),
+        create_template(services[1])
+    ]
+
+    create_ft_billing('2022-02-01', templates[0], provider='foo', notifications_sent=1, billable_unit=2)
+    create_ft_billing('2022-02-01', templates[1], provider='foo', notifications_sent=4, billable_unit=8)
+
+    create_ft_billing('2022-02-01', templates[0], provider='bar', notifications_sent=16, billable_unit=32)
+    create_ft_billing('2022-02-01', templates[1], provider='bar', notifications_sent=64, billable_unit=128)
+
+    results = fetch_daily_sms_provider_volumes_for_platform(start_date='2022-02-01', end_date='2022-02-01')
+
+    assert len(results) == 2
+    assert results[0].provider == 'bar'
+    assert results[0].sms_totals == 80
+    assert results[0].sms_fragment_totals == 160
+
+    assert results[1].provider == 'foo'
+    assert results[1].sms_totals == 5
+    assert results[1].sms_fragment_totals == 10
+
+
+def test_fetch_daily_sms_provider_volumes_for_platform_for_platform_calculates_funky_fragments_and_costs(
```

Comment:
```suggestion
def test_fetch_daily_sms_provider_volumes_for_platform_for_platform_calculates_chargeable_units_and_costs(
```

I'm tempted to leave it but I suspect people in the future won't know what we're on about so we should stick to the official term.

Reply:
Optional: you could reasonably combine this and the previous test as the generic `test_fetch_daily_sms_provider_volumes_for_platform` - together they convey what the function returns and it's useful to see it in one place.